app/forms/expenses/travelAndAccommodation/CostsNotCoveredFormProvider.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.expenses.travelAndAccommodation
+
+import forms.{LessThanZeroError, NonNumericError, OverMaxError}
+import forms.mappings.Mappings
+import models.common.{MoneyBounds, UserType}
+import play.api.data.Form
+
+import javax.inject.Inject
+
+class CostsNotCoveredFormProvider @Inject() extends Mappings with MoneyBounds {
```

Comment:
@vfryer12 
The service uses generic CurrencyFormProvider; please check if it suits the requirements for currency fields.

Reply:
Updated to CurrencyFormProvider